syntax ="proto2";

message Response
{
	required string status=1;
	required string pVersion=2 [default="Version: 1.0"];
	required string url=3;
	required string sInfo=4 [default="Version: 1.0"];
	required string encoding=5 [default="utf-8"];
	required string content=6;
	required string signature=7;
}